docs/foundations/oidc-cheet-sheet.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: What is OpenID Connect?
+id: oidc
+description: 'An introduction to OpenID Connect.'
+slug: /oidc
+keywords:
+  - oidc
+  - oauth
+  - authentication
+  - authorization
+pagination_next: null
+pagination_prev: null
+last_update:
+  date: 09/29/2023
+  author: Anna Garcia
+draft: false
+doc_type: concept
+displayed_sidebar: mainSidebar
+---
+
+## Overview
+
+[OpenID Connect (OIDC)](/docs/terms/oidc) is an identity layer built on top of the [OAuth 2.0 framework](/docs/terms/oauth). While OAuth 2.0 helps with _authorization_ (allowing access), OIDC helps with _authentication_ (identification). OIDC enables your application to verify an end user based on authentication performed by the OIDC Provider's Authorization Server as well as to obtain basic information about the end user. Together, OAuth and OIDC provide a comprehensive solution for secure authentication and authorization used by many identity providers (think "Sign in with Google").
```

Comment:
It might be good to include in the overview that the main thing OIDC adds is the id_token. For example, "OIDC adds the id_token to OAuth's access_token and refresh_token..." or similar.

Reply:
done!